### DISCUSSION TITLE
blog/mencoba-next-14

### DISCUSSION THREAD

**Discussion by giscus** [hendraaagil/website#58] [Comments]
# blog/mencoba-next-14

Pengalaman ketika menggunakan Next.js 14.

https://hendraaagil.dev/blog/mencoba-next-14

**Comment by startdz** [comment]
Coba dong bahas mengenai external redirects url bang, saya masih stuck dibagian ini🥹

**Reply by hendraaagil** [reply]
Stuck gimana bang @startdz ?
